diff --git a/node_modules/pdfjs-dist/build/pdf.worker.js b/node_modules/pdfjs-dist/build/pdf.worker.js
index ac3d027..e95bd23 100644
--- a/node_modules/pdfjs-dist/build/pdf.worker.js
+++ b/node_modules/pdfjs-dist/build/pdf.worker.js
@@ -10660,7 +10660,7 @@ class PartialEvaluator {
           glyphWidth = glyph.vmetric ? glyph.vmetric[0] : -glyphWidth;
         }
         let scaledDim = glyphWidth * scale;
-        if (category.isWhitespace) {
+        /*if (category.isWhitespace) {
           if (!font.vertical) {
             charSpacing += scaledDim + textState.wordSpacing;
             textState.translateTextMatrix(charSpacing * textState.textHScale, 0);
@@ -10670,7 +10670,7 @@ class PartialEvaluator {
           }
           saveLastChar(" ");
           continue;
-        }
+        }*/
         if (!category.isZeroWidthDiacritic && !compareWithLastPosition(scaledDim)) {
           if (!font.vertical) {
             textState.translateTextMatrix(scaledDim * textState.textHScale, 0);
@@ -10696,9 +10696,9 @@ class PartialEvaluator {
           textChunk.prevTransform = getCurrentTextTransform();
         }
         const glyphUnicode = glyph.unicode;
-        if (saveLastChar(glyphUnicode)) {
+        /*if (saveLastChar(glyphUnicode)) {
           textChunk.str.push(" ");
-        }
+        }*/
         textChunk.str.push(glyphUnicode);
         if (charSpacing) {
           if (!font.vertical) {
